src/authutil.ts

```diff
@@ -25,6 +25,10 @@ export function configAuthentication(
   writeFeedToFile(feedUrl, existingNuGetConfig, tempNuGetConfig);
 }
 
+function isValidKey(key: string): boolean {
+  return /^[0-9\w\-\_\.]+$/i.test(key);

```

Comment:
nitpick: \w - Equivalent to [A-Za-z0-9_]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes

Reply:
Thank you!